src/basic/process-util.c

```diff
@@ -2107,7 +2107,7 @@ int posix_spawn_wrapper(
 
                 return FLAGS_SET(flags, POSIX_SPAWN_SETCGROUP);
         }
-        if (ERRNO_IS_NOT_SUPPORTED(r)) {
+        if (ERRNO_IS_NOT_SUPPORTED(r) || r == E2BIG) {
```

Comment:
Please add a short comment what `E2BIG` means here. Not only in the commit message.

Reply:
Thanks, pushed a new version that includes a comment as well.